main.py

```diff
@@ -76,7 +76,8 @@
                     with set_session(app, session):
                         # Start the Hydrus app
                         http_server = WSGIServer(('', 8080), app)
-                        print("Server running")
+                        print("Server running at:")
+                        print("http://localhost:" + str(http_server.server_port) + "/" + API_NAME)
```

Comment:
The server won't always run on localhost. You should use combination of `HYDRUS_SERVER_URL` (it contains both the server IP and port) and `API_NAME`.